tests/karma/all.conf.js

```diff
@@ -82,16 +84,15 @@ module.exports = function (config) {
 		// available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
 		customLaunchers: {
 			bigScreen: {
-				base: "Chrome",
-				flags: ["--window-size=800,800"]
```

Comment:
Why do you remove "--widndow-size", is it not supported?

Reply:
I thought it isn't but apparently it is supported: 
https://stackoverflow.com/questions/43541925/how-to-set-the-browser-window-size-when-using-google-chrome-headless
https://peter.sh/experiments/chromium-command-line-switches/#window-size

Reply:
It's back